### DISCUSSION TITLE
[Board]: Z-Power Management

### DISCUSSION THREAD

**Discussion by vszander** [meshtastic/firmware#4733] [General]
### SOC

ESP32

### Lora IC

HelTec WiFi LoRa 32 (V3)

### Product Link

https://wx4tc.com/?page_id=2792

### Description

This is being rapidly prototyped.   It is NOT  a new  LoRA / microcontroller... the prototype has a plug-in for the HelTec WiFi LoRa 32 (V3).    It is intended to provide high-power ( 12v, 20A ) control circuits  as well as manage multiple batteries ( 2.5Ah on-board) via the meshtastic framework.  Meshtastic developers will enjoy having a supported LoRa enabled microcontroller already contained in a compact battery / management solution  to power their projects.   The original use case was:  power an emergency radio (ham radio in particular) repeater ... but be able to manage power and monitor battery health remotely (LoRA).     I could use help / guidance from the community on how we can leverage the serial-parallel (  74LS595 ) as a module.

**Comment by code8buster** [comment]
74HC165 shift registers are being used in this pull request for Chatter 2.0 as a keyboard input:
https://github.com/meshtastic/firmware/pull/4022/files

Hope this helps you glean some idea for your own implementation.
